doc/major_minor_release_instructions.md

```diff
@@ -43,75 +43,95 @@ make regenerate-test-deploy-manifests
 ### Write up the extract/restore functionality
 
 Every extract/restore is a bit different, so there's no one-size-fits-all
-instructions here. But if you were, e.g. just released 1.11 and are updating
-master to be 1.12, you'd do the following:
+instructions here. But if you were, e.g. just released 1.12 and are updating
+master to be 2.0, you'd do the following:
 
 #### Update protos
 
 Make a copy of the protos:
 
 ```shell
-mkdir -p src/client/admin/v1_11/pfs
-cp src/client/pfs/pfs.proto src/client/admin/v1_11/pfs/pfs.proto
-mkdir -p src/client/admin/v1_11/pps
-cp src/client/pps/pps.proto src/client/admin/v1_11/pps/pps.proto
-mkdir -p src/client/admin/v1_11/auth
-cp src/client/auth/auth.proto src/client/admin/v1_11/auth/auth.proto
+mkdir -p src/client/admin/v1_12/pfs
+cp src/client/pfs/pfs.proto src/client/admin/v1_12/pfs/pfs.proto
+mkdir -p src/client/admin/v1_12/pps
+cp src/client/pps/pps.proto src/client/admin/v1_12/pps/pps.proto
+mkdir -p src/client/admin/v1_12/auth
+cp src/client/auth/auth.proto src/client/admin/v1_12/auth/auth.proto
+mkdir -p src/client/admin/v1_12/enterprise
+cp src/client/enterprise/enterprise.proto src/client/admin/v1_12/enterprise/enterprise.proto
 ```
 
 Then, in the copied over protos:
 
 * Change the package names to be suffixed with the version, e.g. in
-  `pfs.proto` change `package pfs`; to `package pfs_1_11;`. Also update the
+  `pfs.proto` change `package pfs`; to `package pfs_1_12;`. Also update the
   `go_package` field.
 * Update the imports to reference the copied protos. e.g. in `pps.proto`,
   change `import client/pfs/pfs.proto` to
-  `import client/admin/v1_11/pfs/pfs.proto`, and any uses of `pfs.` to
-  `pfs_1_11.`.
+  `import client/admin/v1_12/pfs/pfs.proto`, and any uses of `pfs.` to
+  `pfs_1_12.`.
 
 In `src/client/admin/admin.proto`:
 
 * Import the copied PFS and PPS protos
-* Rename `Op1_11` (which should already exist) to `Op1_12`.
-* Add a new `Op1_11`, which is basically a copy of `Op1_12` referencing the
+* Rename `Op1_12` (which should already exist) to `Op2_0`.
+* Add a new `Op1_12`, which is basically a copy of `Op2_0` referencing the
```

Comment:
Nit: should this be "add a new `Op2_0`, which is basically a copy of `Op1_12`"?

Reply:
It was confusing to me as well. The "Add" step is after rename. So Op1_12 is now Op2_0